cf7-pf_oli_klay/templates/frontend/inputs.php

```diff
@@ -32,10 +32,10 @@
             <select name="fonds" id="fonds" class="input-field">
                 <option value="humanitaire" <?php echo ($_SESSION["fund_code"] == "humanitaire") ? 'selected' : '' ?>><?php _e('Aktuelle Nothilfe', 'donation-form'); ?></option>
                 <option value="noel" <?php echo ($_SESSION["fund_code"] == "noel") ? 'selected' : '' ?>><?php _e('Weihnachtsgeschenk', 'donation-form'); ?></option>
-                <option value="wash" <?php echo ($_SESSION["fund_code"] == "wash") ? 'selected' : '' ?>><?php _e('Sauberes Wasser', 'donation-form'); ?></option>
-                <option value="csp" <?php echo ($_SESSION["fund_code"] == "csp") ? 'selected' : '' ?>><?php _e('Babys und Müttern helfen', 'donation-form'); ?></option>
-                <option value="sansparrain" <?php echo ($_SESSION["fund_code"] == "sansparrain") ? 'selected' : '' ?>><?php _e('Kinder, die ihren Paten verloren haben', 'donation-form'); ?></option>
+                <option value="eau" <?php echo ($_SESSION["fund_code"] == "eau") ? 'selected' : '' ?>><?php _e('Sauberes Wasser', 'donation-form'); ?></option>
+                <option value="survie" <?php echo ($_SESSION["fund_code"] == "survie") ? 'selected' : '' ?>><?php _e('Babys und Müttern helfen', 'donation-form'); ?></option>
                 <option value="medical" <?php echo ($_SESSION["fund_code"] == "medical") ? 'selected' : '' ?>><?php _e('Medizinische Hilfe', 'donation-form'); ?></option>
+                <option value="sansparrain" <?php echo ($_SESSION["fund_code"] == "sansparrain") ? 'selected' : '' ?>><?php _e('Kinder, die ihren Paten verloren haben', 'donation-form'); ?></option>
```

Comment:
I don't think those changes are correct, can you check @numerosept ?

Reply:
Either these are incorrect
or
the links on https://devcomp.site/donations/ are incorrect

I've changed these to match the links on this page, otherwise the select wouldn't apply correctly

Reply:
yes it's not correct, here are the correct options : 
<select name="fonds" id="fonds" class="input-field">
                <option value="humanitaire" <?php echo ($_SESSION["fund_code"] == "humanitaire") ? 'selected' : '' ?>><?php _e('Aktuelle Nothilfe', 'donation-form'); ?></option>
                <option value="noel" <?php echo ($_SESSION["fund_code"] == "noel") ? 'selected' : '' ?>><?php _e('Weihnachtsgeschenk', 'donation-form'); ?></option>
                <option value="wash" <?php echo ($_SESSION["fund_code"] == "wash") ? 'selected' : '' ?>><?php _e('Sauberes Wasser', 'donation-form'); ?></option>
                <option value="csp" <?php echo ($_SESSION["fund_code"] == "csp") ? 'selected' : '' ?>><?php _e('Babys und Müttern helfen', 'donation-form'); ?></option>
                <option value="sansparrain" <?php echo ($_SESSION["fund_code"] == "sansparrain") ? 'selected' : '' ?>><?php _e('Kinder, die ihren Paten verloren haben', 'donation-form'); ?></option>
                <option value="medical" <?php echo ($_SESSION["fund_code"] == "medical") ? 'selected' : '' ?>><?php _e('Medizinische Hilfe', 'donation-form'); ?></option>
                <option value="bible" <?php echo ($_SESSION["fund_code"] == "bible") ? 'selected' : '' ?>><?php _e('Bibelfonds', 'donation-form'); ?></option>
            </select>

Reply:
`<select name="fonds" id="fonds" class="input-field">
                <option value="humanitaire" <?php echo ($_SESSION["fund_code"] == "humanitaire") ? 'selected' : '' ?>><?php _e('Aktuelle Nothilfe', 'donation-form'); ?></option>
                <option value="noel" <?php echo ($_SESSION["fund_code"] == "noel") ? 'selected' : '' ?>><?php _e('Weihnachtsgeschenk', 'donation-form'); ?></option>
                <option value="wash" <?php echo ($_SESSION["fund_code"] == "wash") ? 'selected' : '' ?>><?php _e('Sauberes Wasser', 'donation-form'); ?></option>
                <option value="csp" <?php echo ($_SESSION["fund_code"] == "csp") ? 'selected' : '' ?>><?php _e('Babys und Müttern helfen', 'donation-form'); ?></option>
                <option value="sansparrain" <?php echo ($_SESSION["fund_code"] == "sansparrain") ? 'selected' : '' ?>><?php _e('Kinder, die ihren Paten verloren haben', 'donation-form'); ?></option>
                <option value="medical" <?php echo ($_SESSION["fund_code"] == "medical") ? 'selected' : '' ?>><?php _e('Medizinische Hilfe', 'donation-form'); ?></option>
                <option value="bible" <?php echo ($_SESSION["fund_code"] == "bible") ? 'selected' : '' ?>><?php _e('Bibelfonds', 'donation-form'); ?></option>
            </select>
`